src/clients/users_auto.rb

```diff
@@ -129,13 +129,8 @@ def main
         @ret = Users.Read == ""
         Progress.set(@progress_orig)
       elsif @func == "Write"
-        Yast.import "Progress"
-        Users.SetWriteOnly(true)
-        @progress_orig = Progress.set(false)
-        @ret = Users.Write == ""
-        Progress.set(@progress_orig)
-      # Return if configuration  was changed
-      # return boolean
+        # NOTE: this code was moved to users_finish client, as users set up is
+        # done now in the 1st stage.
```

Comment:
Hm, I would be carefully here because we have a use case in "yast2 autoyast" module where you can change settings and "apply" to the system via the write calls. Another use case is "yast ayast_setup" which is used in a running system. This functionality is mainly used by the SAP team. But I am not sure if they are using the user module too.